Framework/Kernel/src/Atom.cpp

```diff
@@ -1,5 +1,6 @@
 #include <algorithm>
 #include <sstream>
+#include <stdexcept>
```

Comment:
Oh yes, the gcc 4.9 problem. I should strictly say yes but they're not going to cause any problems so I'm happy they stay in here.